lib/rubocop/monkey_patches/rescue_ensure_alignment.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+module RuboCop
+  module Cop
+    module Layout
+      class RescueEnsureAlignment < Base
+        # @TODO remove this monkeypatch after we upgrade from RuboCop 0.91.0
+        def begin_end_alignment_style
+          # FIXME: Workaround for pending status for `Layout/BeginEndAlignment` cop
+          #        When RuboCop 1.0 is released, please replace it with the following condition.
```

Comment:
Could this file print a warning if it's used with rubocop > this version or something?